pennylane/__init__.py

```diff
@@ -102,9 +102,11 @@
 
 import pennylane.operation
 import pennylane.expval
-import pennylane.template
+
+import pennylane.templates
 from pennylane.about import about
```

Comment:
Can we add imports to the necessary places so that the user can do the following:

```import pennylane as qml

qml.init.some_init_funtion(...)
qml.templates.embeddings.some_embedding(...)
qml.templates.layers.some_layer(...)
```

Currently `qml.init`, `qml.templates.embeddings` and `qml.templates.layers` do not find the appropriate modules

Reply:
Added three modules layers, embeddings and init to __init__ file and fixed resulting import problems